cpp/ql/test/TestUtilities/dataflow/FlowTestCommon.qll

```diff
@@ -25,7 +25,12 @@ class IRFlowTest extends InlineExpectationsTest {
     exists(IRDataFlow::Node source, IRDataFlow::Node sink, IRDataFlow::Configuration conf, int n |
       tag = "ir" and
       conf.hasFlow(source, sink) and
-      n = strictcount(IRDataFlow::Node otherSource | conf.hasFlow(otherSource, sink)) and
+      n =
+        strictcount(int line, int column |
+          conf.hasFlow(any(IRDataFlow::Node otherSource |
+              otherSource.hasLocationInfo(_, line, column, _, _)
+            ), sink)
```

Comment:
Do I read this correctly as counting the number of (line, column)-pairs?

Reply:
Yep

Reply:
Shouldn't this (and the other case like it) check the `File` matches as well?  We do sometimes have multiple files in a test so it seems like there's a risk of collision.